### DISCUSSION TITLE
going to voice settings shouldn't immediately prompt mic access

### DISCUSSION THREAD

**Discussion by toscompliantname** [stoatchat/discussions#112] [Feature Suggestions]
Even if I go on the https://app.revolt.chat/settings/audio page in the web app, I don't want to be prompted to turn my mic on just yet - I'm just looking around, exploring what can be configured. 
The way things are now is invasive (I'm thinking about the block or allow pop-up the browsers do when prompted for access) and not very nice. I say make prompting for mic access one additional click away from just going to the "Voice settings" tab.

**Comment by insertish** [comment]
Opened an issue here: https://github.com/revoltchat/revite/issues/242
